rllib/evaluation/multi_agent_sample_collector.py

```diff
@@ -1,249 +0,0 @@
-import logging
```

Comment:
Remove old prealloc-buffer style solutions (not feasible for now; will revisit special cases later, where this type of solution should be faster).